### PR TITLE
Improve help text

### PR DESCRIPTION
**Pull request by fishnix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,16 @@ The JSON control file has the format:
 
 It's possible it could do more than just filter in the future.
 
+`deco` can currently source it's JSON control file from:
```
Comment (dgoerger) [inline README.md]: `its` :P

**Review by dgoerger — COMMENTED** [review]
lgtm except for grammar nit :+1:

**Comment by tenyo** [comment]
👍

File: README.md (hunk 1 above)
Comment (fishnix) [reply README.md]: hey!  I thought i was the grammar nanny!

**Pull request merged**